packages/terra-form-select/src/_Option.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './_Option.scss';
+
+const cx = classNames.bind(styles);
+
+/* Disabling validation for controlled props. These props will always be overwritten. */
```

Comment:
This prevents them from showing up in the props table, I wasn't able to find an existing strategy to hide these from the public API. The parent of these will always override them during cloning, so it will never honor a custom users value if one is provided so I didn't want to expose them in the readme